ladybug/comfort/standard/pmv.py

```diff
@@ -0,0 +1,406 @@
+        return self._heat_loss_convection
+
+    def ToString(self):
+        """Overwrite .NET ToString."""
+        return self.__repr__()
+
+    def __repr__(self):
+        """PMV comfort model representation."""
+        return "PMV Comfort Model: {} values".format(self._calc_length)
+
+
+class PMVComfortParameters(ComfortParameter):
+    """Parameters of PMV comfort."""
+
+    def __init__(self, ppd_comfort_thresh=None, humid_ratio_upper=None,
+                 humid_ratio_lower=None, still_air_threshold=None):
+        """Initalize PMVComfortParameters
+
+        Properties:
+            ppd_comfort_thresh
+            humid_ratio_upper
+            humid_ratio_low
+            still_air_threshold
+        """
+
+        self._ppd_thresh = ppd_comfort_thresh or 10
+        self._hr_upper = humid_ratio_upper or 1
+        self._hr_lower = humid_ratio_lower or 0
+        self._still_thresh = still_air_threshold or 0.1
+
+        assert 5 <= self._ppd_thresh <= 100, \
+            'ppd_comfort_thresh must be between 5 and 100. Got {}'.format(
+                self._ppd_thresh)
+        assert 0 <= self._hr_upper <= 1, \
+            'humid_ratio_upper must be between 0 and 1. Got {}'.format(
+                self._hr_upper)
+        assert 0 <= self._hr_lower <= 1, \
+            'humid_ratio_lower must be between 0 and 1. Got {}'.format(
+                self._hr_lower)
+        assert 0 <= self._still_thresh, \
+            'still_air_threshold must be greater than 0. Got {}'.format(
+                self._still_thresh)
+        assert self._hr_lower <= self._hr_upper, \
+            'humid_ratio_lower must be less than humid_ratio_upper. {} > {}'.format(
+                self._hr_lower, self._hr_upper)
+
+    @property
+    def ppd_comfort_thresh(self):
+        """The threshold of the percentage of people dissatisfied (PPD) beyond which
+        the conditions are not acceptable.  The default is 10%.
+        """
+        return self._ppd_thresh
+
+    @property
+    def humid_ratio_upper(self):
+        """A number representing the upper boundary of humidity ratio above which conditions
+        are considered too humid to be comfortable.  The default is set to 0.03 kg
+        wather/kg air.
+        """
+        return self._hr_upper
+
+    @property
+    def humid_ratio_lower(self):
+        """A number representing the lower boundary of humidity ratio below which conditions
+        are considered too dry to be comfortable.  The default is set to 0 kg wather/kg
+        air.
+        """
+        return self._hr_lower
+
+    @property
+    def still_air_threshold(self):
+        """A number representing the wind speed beyond which the formula for Standard
+        Effective Temperature (SET) is used to dtermine PMV/PPD (as opposed to Fanger's
+        original equation). The default is set to 0.1 m/s.
+        """
+        return self._still_thresh
+
+    def is_comfortable(self, ppd, humidity_ratio):
```

Comment:
This is a method that I would expect to get from PMV class itself and not PMVParameters.

Reply:
There is no comfort model that actually tells you if people will be comfortable or if a given set of conditions are acceptable.  The PMV model only tells you what the mean vote should be across the population, UTCI only tells you the "feels like" temperature.  The judgement about whether a given level of PMV or UTCI are acceptable comes from committees and standards, such as the ASHRAE-55 standards (hence the name of this sub-module, standard).  These are wholly separate from the comfort models themselves and the scientists who made the models.  Furthermore, these standards different depending on building type (for example, some buildings have an acceptable PPD of 20% instead of the usual 10%).  These standards can also change from year to year depending on how the committee is feeling.  This is why this is separated out into a separate module.

Reply:
Let me put it this way. When you calculate UTCI for a point in time and you want to know which condition of person this value represents where would you expect to find the method to evaluate that? Under UTCI class or under UTCI parameters? Just from API design point of view.

Reply:
Yes, we have agreed that we will do it this way for now and, if it becomes confusing in the future, the methods for checking comfort can be moved to be standalone methods.  However, I don't think many people will want to use these methods since I imagine that most people will be trying to estimate thermal comfort for datacollections and there will be special objects to handle those.